src/main/java/rpgcharacters/userflow/CreateUserMenu.java

```diff
@@ -19,20 +17,28 @@ public CreateUserMenu (Scanner sc) {
     }
 
     public boolean createUser (String user, String pass) {
-
-        // TODO: Modify to use SQL and have specific printouts for cases:
-        // - Username already in use
-        // - Other error occured
-        //
-        // Return true if no errors and use is created; False otherwise.
-
-        return true;
+        try {
+            String query = "INSERT INTO user VALUES ("
+                         + "'" + user.replaceAll("'", "''") + "',"
+                         + "'" + pass.replaceAll("'", "''") + "');";
+            Statement stmt = conn.createStatement();
+            stmt.execute(query);
+            return true;
+        } catch (SQLException e) {
+            if (e.getMessage().startsWith("Unique index or primary key violation")) {
+                System.out.println("\nUser already exists!\n");
+            } else {
```

Comment:
It'll create the user with empty strings...

We should do this check on the db level though and not through our app. That should be a separate PR so I'll create an issue for it and work on it after I finish with the SQL